apps/experiments/migrations/0039_update_AWS_synthetic_voices.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2 on 2023-10-11 15:18
+import os, json
+from django.db import migrations
+
+
+def load_language_code(apps, schema_editor):
+    SyntheticVoice = apps.get_model("experiments", "SyntheticVoice")
+    voice_data = {}
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(current_directory, "preload_data/aws_voices.json")
+
+    with open(file_path, "r") as json_file:
+        voice_data = json.load(json_file)["voices"]
+
+    voices_created = 0
+    voices_edited = 0
+    for voice in voice_data:
+        _, created = SyntheticVoice.objects.update_or_create(
+            name=voice["name"],
+            language=voice["language"],
+            gender=voice["gender"],
+            neural=voice["neural"],
+            service=voice["service"],
+            defaults={
```

Comment:
this should be in the `defaults` since it's a new field:
```suggestion
            defaults={
                "service": voice["service"],
```

Reply:
Also should have mentioned this was done in https://github.com/dimagi/open-chat-studio/pull/48/commits/28e0bf3ff7a2a3849a56526089598781a88ee31c